### ISSUE TITLE
Missing a skipnav link for the main navigation bar

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4105]
## Area or Component
Main navigation bar on the internal application

## Quick Summary
We're missing a "skipnav" link as a keyboard item so users can skip over our navigation bar. Tabbing through the site requires you to go through the navigation links every time.

### How to Replicate
Open up simplereport.gov/app and start tabbing. Notice there's no option to skip navigation and that you have to tab through those elements every time.

## Accessibility Violation
A keyboard-functional "skip" link SHOULD be provided to allow keyboard users to navigate directly to the main content:
https://dequeuniversity.com/class/semantic-structure/within-pages/skip-navigation

## Action Requested
Create a skipnav link at the top of the DOM and give it a tab index=-1, so it's always the first option for keyboard users. (We have a skipnav link on our static site if you need an example.)
